src/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "ux-aspects",
+  "version": "1.1.0-beta.1",
+  "description": "Open source user interface framework for building modern, responsive, mobile big data applications",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/UXAspects/UXAspects.git"
+  },
+  "keywords": [
+    "ux",
+    "aspects",
+    "ui",
+    "framework",
+    "angular",
+    "angularjs",
+    "bootstrap"
+  ],
+  "author": "Ashley Hunter",
+  "license": "Apache-2.0",
```

Comment:
I thought so, but wasn't sure. Looked at all other open source CAF services and they were all using Apache 2.0 license. Do you know where we can check this?